packages/syft/src/syft/core/adp/vectorized_publish.py

```diff
@@ -177,10 +195,16 @@ def publish(
             raise Exception(
                 "Negative budget spend not allowed in PySyft for safety reasons."
                 "Please contact the OpenMined support team for help."
+                "For that you can either:"
+                " * describe your issue on our Slack #support channel. To join: https://openmined.slack.com/"
+                " * send us an email describing your problem at support@openmined.org"
+                " * leave us an issue here: https://github.com/OpenMined/PySyft/issues"
             )
 
         # Step 3: Check if the user has enough privacy budget for this query
         privacy_budget = get_budget_for_user(verify_key=ledger.user_key)
+        print(privacy_budget)
```

Comment:
I think we could add these to the log because we want them most of the time and the data owners probably would too.